designs/carbon-aware.md

```diff
@@ -0,0 +1,197 @@
+# Carbon Aware Karpenter: Optimizing Kubernetes Cluster Autoscaling for Environmental Sustainability
+*Author: [@JacobValdemar](https://github.com/JacobValdemar)*
+
+## Context & Problem
+There is a growing concern about the environmental impact of Kubernetes clusters. Karpenter's opportunities within environmental sustainability is referenced in multiple comments that back [`karpenter-core`'s move to CNCF](https://github.com/kubernetes/org/issues/4258).
+
+I am currently working on my master's thesis in Computer Engineering (Master of Science in Engineering) at Aarhus University located in Denmark. The objective of the thesis is to enable Karpenter to minimize carbon emissions from Kubernetes clusters that run on cloud infrastructure (scoped to AWS).
+
+RFC: https://github.com/aws/karpenter/issues/4630
+
+## Fundamentals of Green Software
+I will try to keep it simple. The reader should be familiar with the following.
+
+A cluster's emissions is made of two elements: embodied emissions and operational emissions. To get the total emissions, one can add them togeather.
+
+- **Embodied carbon emissions**: Manufacturing emissions (CO₂e) amortized over instance lifetime (usually 4 years) divided by how long we use the instance
```

Comment:
Can we clarify that "instance" in this sense refers to a percentage of a physical machine?

Reply:
@jackfrancis Good point. That is totally not obvious when reading the text. When we say instance lifetime we of course mean the lifetime of the physical machine that the instance is part of 👍

Reply:
a7134dd